tests/rules/null-coallesce.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function f($filename) {
+    // ruleid: embed_word
```

Comment:
this should be `// ruleid: test`

and in the rule we should use also the same `id: test`.